testnets/xiontestnet2/chain.json

```diff
@@ -42,21 +42,21 @@
   },
   "codebase": {
     "git_repo": "https://github.com/burnt-labs/xion",
-    "tag": "v18.0.0",
-    "recommended_version": "v18.0.0",
+    "tag": "v19.0.1",
+    "recommended_version": "v19.0.1",
     "language": {
       "type": "go",
       "version": "v1.23"
     },
     "binaries": {
-      "darwin/amd64": "https://github.com/burnt-labs/xion/releases/download/v18.0.0/xiond_18.0.0_darwin_amd64.zip?checksum=sha256:bf8fa7743d7aab33396d9122ab3817224d41bbbcf410424c993b1d25911faee6",
-      "darwin/arm64": "https://github.com/burnt-labs/xion/releases/download/v18.0.0/xiond_18.0.0_darwin_arm64.zip?checksum=sha256:1f8799dc21ff348ca71e25864e12b1b07c2d9be66359eab9c420887178403d9d",
-      "linux/amd64": "https://github.com/burnt-labs/xion/releases/download/v18.0.0/xiond_18.0.0_linux_amd64.zip?checksum=sha256:361ff8194f0f2601fd8a0759eb218b27d62d236618ca7c623481c64ec7c85a8c",
-      "linux/arm64": "https://github.com/burnt-labs/xion/releases/download/v18.0.0/xiond_18.0.0_linux_arm64.zip?checksum=sha256:1f49577bd44ea2a4072d88e6934e124d44594489b76d33922c4a8803db08303c"
+      "darwin/amd64": "https://github.com/burnt-labs/xion/releases/download/v19.0.1/xiond_19.0.1_darwin_amd64.tar.gz?checksum=sha256:f350e2ab0cc08c18acce0d6518bd4f29ac88f1c331eb10d3ca43ec3e74103521",
+      "darwin/arm64": "https://github.com/burnt-labs/xion/releases/download/v19.0.1/xiond_19.0.1_darwin_arm64.tar.gz?checksum=sha256:9c2944de98c54f4e8517259d0782daca4c43900887b7d7c19044ba75b6e2976d",
+      "linux/amd64": "https://github.com/burnt-labs/xion/releases/download/v19.0.1/xiond_19.0.1_linux_amd64.tar.gz?checksum=sha256:5350ccf4fb83f086f772ceebd8a1c3788c2fa64ad1976d92052842e23a1977c3",
+      "linux/arm64": "https://github.com/burnt-labs/xion/releases/download/v19.0.1/xiond_19.0.1_linux_arm64.tar.gz?checksum=sha256:efed79d4b240c6edaa4488b54e7efcbff89313c0f0ed6e1682dab0f1d97315c2"
     },
     "sdk": {
       "type": "cosmos",
-      "version": "v0.50.13"
+      "version": "v0.53.0"
```

Comment:
Path: sdk.version, Reason: Value mismatch, In "codebase": v0.50.13, In currentVersion: v0.53.0